README.md

```diff
@@ -8,172 +8,13 @@ prototyping environment on Heroku.
-prototype-2	prototype-2-prototype
-```
-
-Save the file to the directory you just created, for example, `~/projects/prototypes/apps.tsv`
-
-Assuming you have cloned this repository to `~/projects/hmrc/init-prototype`, run the script as follows,
-
-```shell script
-cd ~/projects/prototypes && \
-~/projects/hmrc/init-prototype/bin/compare-repositories.sh apps.tsv > out.tsv
-```
-
-The output saved to `out.tsv` indicates for each Heroku app, whether the HMRC repository is the same
-as the Heroku repository with TRUE or FALSE. For example,
-
-```text
-herokuApp	hmrcRepository	isSynced
-prototype-1	prototype-1-prototype	TRUE
-prototype-2	prototype-2-prototype	FALSE
-```
-
-## Generate report of deployed repositories with details of GitHub repo, package.json, and package-lock.json
-
-The sbt task `generatePackageLockReport` can produce a report of which Github repos deployed to Heroku contain a 
-`package.json` file and a `package-lock.json` file. To do this, the task will:
-1. Parse a local copy of `report.txt` generated by `generateHerokuReport` (meaning this task must be run as a prerequisite)
-2. For each deployed prototype, check if a Github repo exists with the same name as the Heroku app
-3. If not, try to resolve the Github repo name by searching for the commit ids of the 10 earliest Heroku releases of the app
-4. If the Github repo is known, check if it contains a package.json and/or package-lock.json
-5. Produce a report of deployed Heroku prototypes with the following details:
-* Heroku app name
-* Github repo name
-* If the repo has a package.json
-* If the repo has a package-lock.json
-* If the prototype is deployed to Heroku
-* If the prototype is running in Heroku
-* If the Github repo was found
-
-The output will be saved to `package-lock-analysis.tsv`.
-
-```shell script
-sbt -Dgithub.apiToken=REPLACE_WITH_GITHUB_PERSONAL_ACCESS_TOKEN "generateHerokuReport report.txt"
-sbt generatePackageLockReport
-```
-## Updating repository.yaml files for prototypes where YAML contains `prototype-auto-publish`
-The `updateRepositoryYaml` task will 
-- Parse a local JSON file of prototype repository details
-- Shallow clone the prototypes repositories into a local directory
-- Check to see if the `repository.yaml` contains `prototype-auto-publish` but not `prototype-name`
-- Add a prototype name based on the `prototype-url`
-- Confirm with the user whether to update the files (based on stdin)  
-- Push the change to `main` is possible, or to a branch if cannot merge directly to `main`
-- Write the name of updated repositories in either `pushed-to-main.txt` or `pushed-to-branch.txt`
-
-The JSON can be retrieved from this endpoint:
-https://catalogue.tax.service.gov.uk/api/v2/repositories?archived=false&repoType=Prototype
-
-The current task will expect the JSON file to be named `prototypes-from-catalogue.json`, and will check out to branch
-`PLATUI-2475_bulk-update-repository-yaml` if cannot push to `main`.
-
 ## Local testing of prototype creation
```

Comment:
it might be worth having a list here that summarises what steps the prototype creation task does in Main.scala at a high level